File: nextjs-frontend_store/models/rating/RatingPagingVm.ts
import {RatingVm} from "@/models/rating/RatingVm";

export type RatingPagingVm = {
    ratingsPayload: RatingVm[],
    pageIndex: number,
    pageSize: number,
    totalElements: number,
    totalPages: number,
    isLast : boolean
}

File: feedback/src/main/java/com/lexuancong/feedback/viewmodel/rating/RatingPagingVm.java
package com.lexuancong.feedback.viewmodel.rating;

import java.util.List;

public record RatingPagingVm(
        List<RatingVm> ratingPayload,
        int pageIndex,
        int pageSize,
        int totalElements,
        int totalPages,
        boolean isLast
) {
}
